docs/sql_reference/src/asciidoc/_chapters/olap_functions.adoc

```diff
@@ -461,8 +487,110 @@ FROM persnl.employee;
+
+`LEAD (expression, offset, default-value)`
+
+* `_expression_`
++
+Specifies a scalar expression or a column whose value from the subsequent row is to be returned. If the subsequent row does not exist in the current group, `_expression_` returns NULL.
+
+* `_offset_`
++
+Specifies an expression that can be evaluated to a positive integer indicating the offset after the current row. For example, a value of 1 accesses the next row and a value of 3 accesses the third row from the current row.
+
+* `_default-value_`
++
+Specifies the value if the value returned by _expression_ is NULL.
+
+<<<
+[[examples_of_lead_window_function]]
+=== Examples of LEAD Window Function
+
+The LEAD function in the following query returns the start date of next plan and DATEADD function subtracts one day from that date as the end date of the current plan.
+
+```
+SELECT *,
+DATEADD(DAY, -1, LEAD(StartDate, 1,'01-Jan-2100')
+     OVER (PARTITION BY CustomerCode ORDER BY StartDate ASC)) AS EndDate
+FROM CustomerPlan
+```
+
+<<<
+[[lag_window_function]]
+== LAG Window Function
+
+The LAG window function can be used to access the previous rows from the same result set without using self-joins. The LAG function takes the same three parameters as the LEAD function with the exception that `_offset_` specifies the offset to access a row that comes before the current row. The return type of LAG is that of `_expression_`.
+
+`LAG (expression, offset, default-value)`
+
+* _expression_
++
+Specifies a scalar expression or a column whose value from the previous row is to be returned. If the previous row does not exist in the current group, `_expression_` returns NULL.
+
+* _offset_
++
+Specifies an expression that can be evaluated to a positive integer indicating the offset before the current row. For example, a value of 1 accesses the previous row and a value of 3 accesses the third row before the current row.
+
+* _default-value_
++
+Specifies the value if the value returned by `_expression_` is NULL.
+
+<<<
+[[examples_of_lag_window_function]]
+=== Examples of LAG Window Function
+
+The LAG function in the following query returns the plan code of a previous plan and ’N/A’ for the first plan.
+
+```
+SELECT  CustomerCode, PlanCode AS CurrentPlanCode,
+              LAG(PlanCode, 1, 'NA')
+              OVER (PARTITION BY   CustomerCode ORDER BY StartDate ASC)   AS LastPlan
+FROM   CustomerPlan;
+```
+
+<<<
+[[first_value_window_function]]
+== FIRST_VALUE Window Function
+
+The FIRST_VLAUE window function returns the first value from a sorted partition.
+
+`FIRST_VALUE (expression [ RESPECT NULLS | IGNORE NULLS ] )`
+
+* _expression_
++
+Specifies a scalar expression or a column whose value from the first row in the partition is to be returned.
+
+* IGNORE NULLS
++
+Specifies the function to return first non-null value.

```

Comment:
Thanks Hans. 
Your comment has been implemented. :) 
And as suggested by Ming, "Trafodion does not support RESPECT NULLS & INGORE NULLS, but it only supports the function to return first value including null (default behavior).", so I remove [ RESPECT NULLS | IGNORE NULLS ] from systax and add some details.